src/pds/registry/utils/titan_treks/create_titan_treks_pds4.py

```diff
@@ -0,0 +1,79 @@
+"""Script to build pds4 xml for Titan Treks OGC WMTS layers.
+
+The layers are scraped using the Titan Treks API.
+"""
+import argparse
+from pathlib import Path
+
+import requests
+
+from . import product_service_builder as psb
+
+
+def main():
+    """Main function of script."""
+    # set up command line args
+    parser = argparse.ArgumentParser(description="Customize save pds4 xml labels created for Titan Treks",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("-s",
+                        "--save-xml",
+                        action="store_true",
+                        help="True if you want the xml files to be saved",
+                        default=True)
+    parser.add_argument("-d",
+                        "--destination-directory",
+                        help="Directory to save pds4 xml files",
+                        default="xml_files/product_service")
+    parser.add_argument("-v",
+                        "--verbose",
+                        action="store_true",
+                        help="Verbosity of file creation",
+                        default=False)
+
+    args = parser.parse_args()
+    config = vars(args)
```

Comment:
`config` variable name is a bit misleading. Usually I don't add this line using `vars` function, I just call the arguments `args.save_xml` or `args.destination_directory`.